src/Board.coffee

```diff
@@ -61,6 +61,8 @@ class Board
     COLUMNS: BOARD_COLS
     WIDTH: BOARD_WIDTH
     HEIGHT: BOARD_HEIGHT
+  # HERE -> thinking about adding click disable flag...
+  @pauseCollectCoins = false
```

Comment:
`Board.pauseCollectCoins = false` is the effect of using `@pauseCollectCoins = false` inside the class level property indentation (not within the constructor or another method of the class)

When you use `@pauseCollectCoins` in the `clicked` method of the class, you are saying `this.pauseCollectCoins` which is *not the same as* `Board.pauseCollectCoins`.

And in Game, you use `@board.pauseCollectCoins` which is referring to `this.board.pauseCollectCoins` and not `Board.pauseCollectCoins`

*that* is why I said it has no effect on the functionality.

You can remove the line and the rest of your code will still work the exact same way. Because you are using a conditional which evaluates to false (undefined evals to false when checked as a boolean) and otherwise implicitly adding a new property that is true in the Game code.